nvme.c

```diff
@@ -74,6 +74,14 @@
 struct stat nvme_stat;
 const char *devicename;
 
+#define CMD_NOT_YET_SUPPORTED() {				\
+					if (argc) {		\
+						fprintf(stderr, \
+							"Error: this command is not yet supported in the current version of switchtec-nvme utility.\n");\
+						return -1;	\
+					}			\
+				}
+
```

Comment:
The argc checking seems unnecessary. And the following indentation looks better (and move { to a seperate line)?

```
#define CMD_NOT_YET_SUPPORTED() \
{ \
         fprintf(stderr, \
                 "Error: this command is not yet supported in the current version of switchtec-nvme utility.\n"); \ 
         return -1; \
}
```